src/css/profile/mobile/common/popup.less

```diff
@@ -129,6 +128,23 @@
 		background-color: @color_popup_footer_bg;
 		position: sticky;
 		.LESSui-footer();
+
+		div.ui-li-divider::after {
+			content: "";
+			position: absolute;
+			width: 1 * @px_base;
```

Comment:
You can set the value with '@px_base' only.

Reply:
@HunseopJeong 
I know. Is there a problem?

Reply:
You can use the "@px_base" only not the "1 * @px_base".

Reply:
done.